committeeoversightapp/static/committeeoversightapp/js/category_slug.js

```diff
@@ -0,0 +1,9 @@
+$(function() {
```

Comment:
We need Javascript and a [Wagtail hook](https://docs.wagtail.io/en/v2.6.1/reference/hooks.html) to dynamically update the page's slug and SEO title 

The method is based on this blog post, which I've also dropped into a comment in `models.py`: https://timonweb.com/tutorials/how-to-hide-and-auto-populate-title-field-of-a-page-in-wagtail-cms/